docs/sources/alerting/set-up/provision-alerting-resources/_index.md

```diff
@@ -24,25 +24,18 @@ There are three options to choose from:
 
    For more information on the Alerting Provisioning HTTP API, refer to [Alerting provisioning API](https://grafana.com/docs/grafana/latest/developers/http_api/alerting_provisioning/).
 
-1. Provision your alerting resources using Terraform.
+1. Provision your alerting resources using [Terraform](https://www.terraform.io/).
 
 **Note:**
 
-Currently, provisioning for Grafana Alerting supports alert rules, contact points, mute timings, and templates. Provisioned alerting resources using file provisioning or Terraform can only be edited in the source that created them and not from within Grafana or any other source. For example, if you provision your alerting resources using files from disk, you cannot edit the data in Terraform or from within Grafana.
+Currently, provisioning for Grafana Alerting supports alert rules, contact points, notification policies, mute timings, and templates. Provisioned alerting resources using file provisioning or Terraform can only be edited in the source that created them and not from within Grafana or any other source. For example, if you provision your alerting resources using files from disk, you cannot edit the data in Terraform or from within Grafana.
 
-To allow editing of provisioned resources in the Grafana UI, add the `X-Disable-Provenance` header to the following requests in the API:
-
-- `POST /api/v1/provisioning/alert-rules`
```

Comment:
This section was moved to `view-provisioned-resources` since it's very closely related. Maybe we consider renaming that page?

Reply:
To Edit provisioned resources?

Reply:
"View and edit" maybe? The gist is:
- You can never edit file provisioned resources
- You can edit API-provisioned resources by jumping through some hoops